_posts/2023-06-08-opensearch-project-newsletter-volume1-issue4.md

```diff
@@ -0,0 +1,79 @@
+## What’s New?
+
+**[OpenSearch 2.8.0 is here](https://opensearch.org/blog/opensearch-2.8.0-released/),** and it has a host of new features and enhancements and experimental functionality that are sure to generate excitement within the community! This release includes new tools for machine learning, search, and observability workloads and several enhancements to better manage your indexes and data. [Check out what’s new in 2.8.0](https://opensearch.org/blog/opensearch-2.8.0-released/). 
+
+[**Data Prepper 2.3.0 is now available for download!**](https://opensearch.org/downloads.html#data-prepper) This release introduces a number of changes that improve Data Prepper’s ability to process complex expressions with functions, arithmetic operations, and string operations. [Get started with it now.](https://opensearch.org/downloads.html#data-prepper)  
+
+[**OpenSearch 1.3.10**](https://opensearch.org/versions/opensearch-1-3-10.html) is now available and includes bug fixes, infrastructure, enhancements, & maintenance updates. See the [release notes](https://github.com/opensearch-project/opensearch-build/blob/main/release-notes/opensearch-release-notes-1.3.10.md) for full details.
+
+[**Searchable snapshots are now available.**](https://opensearch.org/blog/searchable-snapshots/) With searchable snapshots, you can search indexes that are stored as snapshots within remote repositories in real time without the need to restore the index data to local storage ahead of time. [Download OpenSearch 2.8](https://opensearch.org/downloads.html) to try them out, or read more about [Searchable snapshots](https://opensearch.org/docs/latest/tuning-your-cluster/availability-and-recovery/snapshots/searchable_snapshot/) in the OpenSearch documentation. 
+
+[**Call for presentations: Bring your ideas to OpenSearchCon!**](https://opensearch.org/opensearchcon2023-cfp.html) OpenSearchCon 2023 is now accepting submissions. This year’s conference is happening Sept. 28 in Seattle, with session tracks reserved for search, analytics, observability, and security, and community presentations. Submissions close June 16, so don’t wait to [build your proposal](https://airtable.com/shrVZLtXJ2d1cI6pT)!
+
+**[Take our customer survey and get a OpenSearch tumbler](https://amazonmr.au1.qualtrics.com/jfe/form/SV_1BxJNrtCo4LbweW).**  Each quarter the OpenSearch Project issues a survey to better understand our community members. In the second quarter of 2023, we are collecting data on a UX health metric that will help us understand and quantify the user journey. If your interested and would like to share your experiences, we’re offering an OpenSearch mug in appreciation of your time. Please [use this link to take the survey](https://amazonmr.au1.qualtrics.com/jfe/form/SV_1BxJNrtCo4LbweW), or or you can access it at [OpenSearch.org](http://opensearch.org/). 
```

Comment:
Take our **customer** survey and get a OpenSearch tumbler  
should be 
Take our **community** survey and get a OpenSearch tumbler